astrodendro/dendrogram.py

```diff
@@ -463,6 +468,23 @@ def __eq__(self, other):
         if not (self.data == other.data).all():
             return False
 
+        if self.params['min_value'] != other.params['min_value']:
+            return False
+
+        self_params = self.params
+        other_params = other.params
+
+        self_params.pop('min_value')
+        other_params.pop('min_value')
+
+        # Don't compare params if one has not been set (ie. = 0)
```

Comment:
I don't like this comment, since I think someone could explicitly set a param to 0. I'm fine with the logic since it's getting the tests to pass and 0 might *usually* mean "not explicitly set", but this comment should indicate that this is a small hack/workaround